src/cli/commands/ls.js

```diff
@@ -45,7 +45,7 @@ export async function buildTree(
   const treesByKey = {};
   const trees = [];
   const hoisted = await linker.getFlatHoistedTree(patterns);
-
+  //console.log(hoisted);
```

Comment:
Indeed! When I fully understand the use case of `yarn ls` that I mentioned in #726, I'll add tests, remove this log, and rebase